packages/forms/resources/views/components/wizard/step.blade.php

```diff
@@ -12,6 +12,17 @@
 @endphp
 
 <div
+    x-init="
+        const allChildren = $el.getElementsByTagName('*')
+
+        for (const el of allChildren) {
+            if (el.hasAttribute('autofocus')) {
+                return;
+            }
+        }
```

Comment:
How about using `$el.querySelector('[autofocus]')` to check if there are any elements with that attribute instead of fetching everything and looping through

Reply:
Lol, you know what, that sounds smarter.

Done.